lib/linux/background-managers/cinnamon.js

```diff
@@ -0,0 +1,40 @@
+'use strict';
+
+const childProcess = require('child_process');
+const util = require('util');
+const {existCommand} = require('../util');
+
+const execFile = util.promisify(childProcess.execFile);
+const exec = util.promisify(childProcess.exec);
```

Comment:
Move these to the util file too so we don't have to promisify the child process methods in each desktop manager file.